tests/config/test_config.py

```diff
@@ -30,104 +27,112 @@ def generate_validator_testcases(valid):
 
     validation_tests = (
         {
-            'validator':
-            _listify_validator(validate_float, n_items=2),
-            'success':
-            ((_, [1.5, 2.5])
-             for _ in ('1.5, 2.5', [1.5, 2.5], [1.5, 2.5], (1.5, 2.5),
-                       np.array((1.5, 2.5)))),
-            'fail': ((_, ValueError) for _ in ('fail', ('a', 1), (1, 2, 3)))
+            "validator": _listify_validator(validate_float, n_items=2),
+            "success": (
+                (_, [1.5, 2.5])
+                for _ in (
+                    "1.5, 2.5",
+                    [1.5, 2.5],
+                    [1.5, 2.5],
```

Comment:
Same test twice?

Reply:
Looks like it, can we fix that in a separate issue?